src/components/PriceChart/PriceChartTooltip.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react'
+import { Network } from '@dcl/schemas/dist/dapps/network'
+import { formatAndRoundPriceString } from './utils'
```

Comment:
Would you move this import to the bottom? 🙏

Reply:
done!